modules/n1ql/pages/n1ql-language-reference/reservedwords.adoc

```diff
@@ -193,83 +207,97 @@ The following keywords are reserved and cannot be used as unescaped identifiers:
 | POOL
 | xref:n1ql-language-reference/window.adoc#window-frame-extent[PRECEDING]
 | PREPARE
+| PREV
+| PREVIEW
```

Comment:
Sorry, this should be `PREVIOUS`, not <s>PREVIEW</s>. Again, my bad, can't read my own writing.